crash_similarity.py

```diff
@@ -70,6 +70,18 @@ def get_stack_trace_from_crashid(crash_id):
 def get_stack_traces_for_signature(fnames, signature):
     traces = set()
 
+    #query stack traces online
+    url = 'https://crash-stats.mozilla.com/api/SuperSearch'
+    params = {
+        'signature': '=' + signature,
+        '_columns':  'proto_signature'
+    }
```

Comment:
I am trying using
 `params = {
        'signature': '=' + signature,
        '_facets':  ['proto_signature']
    }`
but the response doesn't contain proto_signature